db/migrate/20151204170038_change_location_name_to_be_non_nil.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeLocationNameToBeNonNil < ActiveRecord::Migration
+  def up
+    change_column :locations, :name, :string, null: false
```

Comment:
Not super important @cawel but I thought you might like this: `change_column_null`

```ruby
change_column_null(:locations, :name, true)
```